src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -759,7 +751,6 @@ namespace FourPipeBeam {
         Real64 rho;                     // local fluid density
         bool noHardSizeAnchorAvailable; // aid for complex logic surrounding mix of hard size and autosizes
         Real64 cpAir = 0.0;
-        int SolFlag;
```

Comment:
[src/EnergyPlus/HVACFourPipeBeam.cc:762]:(style),[variableScope],The scope of the variable 'SolFlag' can be reduced.